samples/net/eth_native_posix/README.rst

```diff
@@ -0,0 +1,112 @@
+.. _eth-native-posix-sample:
+
+Native Posix Ethernet
+#####################
+
+Overview
+********
+
+The eth_native_posix sample application for Zephyr creates a **zeth** network
+interface to the host system. One can communicate with Zephyr via this network
+interface.
+
+The source code for this sample application can be found at:
+:file:`samples/net/eth_native_posix`.
+
+Building And Running
+********************
+
+To build the eth_native_posix sample application, follow the steps
+below.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/net/eth_native_posix
+   :board: native_posix
+   :conf: <config file to use>
+   :goals: build
+   :compact:
+
+Normally one needs extra priviliges to create and configure the TAP device in
+the host system. Use sudo to execute the Zephyr process with admin privileges,
+like this:
+
+.. code-block:: console
+
+    sudo --preserve-env=ZEPHYR_BASE make run
```

Comment:
Seems this feature came in sudo 1.8.21 ( https://www.sudo.ws/stable.html#1.8.21 ) which is quite new (2017-08-23)

Reply:
Ok, then we could just use -E option here. Fortunately this sudo cmd is only mentioned/used in readme file so no changes needed in the code.

Reply:
Are you going to remove the ``--preserve-env=ZEPHYR_BASE`` here (and just use ``--preserve-env``, and in the command examples below?

Reply:
I was hoping to leave the --preserve-env=ZEPHYR_BASE version as that is the preferred way. But as the older sudo version does not support that I added also a way to run the sudo the old way. I did not touch the other sudo calls in that document as I felt that readers are "smart" enough to understand how to run the command.
I have no big preference with the sudo command parameters thou, so if you think that we should just use --preserve-env here, then I can just send a version using just that.

Reply:
Given the comments above, it seems we should just use ``--preserve-env``. But if you'd like to leave the ``preserve-env=ZEPHYR_BASE`` in, add the note mentioned below to the code example.

Reply:
Another topic on the same line:
You are calling here `make run` instead of just running the binary (`zephyr/zephyr.exe`)
Would it not be better to call the binary directly? (so we don't hide from the user what is happening) (I'm thinking that the user will probably want to run the binary in a debugger, pass options to it, or instrument it. So it would seem better to make it evident to them what is the binary they actually run)